code/datums/mood_events/generic_positive_events.dm

```diff
@@ -171,6 +171,10 @@
 /datum/mood_event/nanite_happiness/add_effects(message)
 	description = "<span class='nicegreen robot'>+++++++[message]+++++++</span>\n"
 
+/datum/mood_event/poppy_pin
+	description = "<span class='nicegreen'>I feel proud to show my remembrance of the many who have died to ensure that I have freedom.</span>\n"
+	mood_change = 0.3
```

Comment:
I don't think 0.3 is a good idea, all over instances of mood_change are integers

Reply:
1 is quite significant for such a simple item

Reply:
none probably, but 1 is literally nothing

Reply:
Alright, changed.